Sources/SourceKitLSP/Swift/CodeActions/ConvertIUOToProperOptional.swift

```diff
@@ -0,0 +1,59 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LanguageServerProtocol
+import SwiftRefactor
+import SwiftSyntax
+
+/// Convert implicitly unwrapped optionals (IUOs) to proper optionals
+@_spi(Testing)
```

Comment:
As far as I can tell, we don’t use this type in tests, so `ConvertIUOToProperOptional` doesn’t need to be `@_spi` or `public` but can be `internal` instead.